manifests/instance.pp

```diff
@@ -440,13 +444,51 @@
       }
     }
 
+    if $xpack {
+      if (($ca_certificate == undef) and ($certificate == undef) and ($private_key == undef)) {
```

Comment:
Shouldn't these be `or` instead of `and` ... or `!= undef` ?

Reply:
Good spot, it would be better to use or.

Reply:
This has been added also